pay-api/src/pay_api/models/payment.py

```diff
@@ -247,6 +250,18 @@ def find_payments_to_consolidate(cls, auth_account_id: str):
 
         return query.all()
 
+    @classmethod
+    def get_today_invoices(cls):
+        """Fetch invoices created on today's date, based on the base query structure."""
+        base_query = TransactionQuery.generate_base_transaction_query()
+
+        today_query = (
+            base_query
+            .filter(Invoice.created_on <= datetime.today())
```

Comment:
`datetime.today`, not a good idea that's local date

Reply:
there are some local dates still in the code, I've tried to get rid of a few of them, but this causes issues at 4 pm